### PR TITLE
Fixed a lot of spelling and grammar mistakes, added some formatting a…

### PR DESCRIPTION
**Pull request by SebastianZ**
I've corrected a bunch of spelling and grammar mistakes within the CSS UI 4 spec. and added some missing references.

Sebastian

### CODE REVIEW

File: css-ui-4/Overview.bs [hunk 1]
```diff
@@ -331,21 +331,21 @@ regardless of color background.
...
-The 'outline' property is a shorthand property,
+The 'outline' property is a shorthand property
 and sets all three of 'outline-style',
 'outline-width',
 and 'outline-color'.
 
 Note: The outline is the same on all sides.
 In contrast to borders,
-there are no ''outline-top'' or ''outline-left'' etc. properties.
+there are no ''outline-top'', ''outline-left'' etc. properties.
```
Comment (tabatkins) [inline css-ui-4/Overview.bs]: Need another comma between outline-left and etc here.

File: css-ui-4/Overview.bs [hunk 2]
```diff
@@ -411,10 +411,10 @@ then the outline is outset from the <a>border edge</a> by that amount.
 to shrink into the border box.
 Both the height and the width of outside of the shape
```
Comment (tabatkins) [inline css-ui-4/Overview.bs]:
```suggestion
Both the height and the width of the outside of the shape
```

File: css-ui-4/Overview.bs [hunk 3]
```diff
@@ -529,7 +529,7 @@ The user agent must allow the user to resize the element
 with no other constraints than what is imposed by
 'min-width', 'max-width', 'min-height', and 'max-height'.
 
-Note: There may be situations where user attempts to resize an element
+Note: There may be situations, in which user attempts to resize an element
```
Comment (tabatkins) [inline css-ui-4/Overview.bs]: no comma

File: css-ui-4/Overview.bs [hunk 4]
```diff
@@ -1740,10 +1740,9 @@ Note: The details of the aliasing mechanism is intentionally left up to the UA.
 Making ''-webkit-user-select'' a shorthand property of 'user-select'
 is known to be an effective approach,
 and new implementers should consider it.
-However UAs supporting ''-webkit-user-select'' as an alias of 'user-select' through other means exist,
-without adverse consequences to compatibility,
-so this specification allows flexibility.
-
+However, UAs supporting ''-webkit-user-select'' as an alias of 'user-select' through other means exist,
+without adverse consequences to compatibility.
+So this specification allows flexibility.
```
Comment (tabatkins) [inline css-ui-4/Overview.bs]: this line should remain part of the previous sentence, as originally written. (the addition of a comma after 'however' is good, tho)

File: css-ui-4/Overview.bs [hunk 5]
```diff
@@ -1794,7 +1793,7 @@ so that CSS can be used to restyle them.
 	and a "none" value that suppresses "native" look.
 
 	However, there is evidence that this alone is not web compatible,
-	due in part to uses such as:
+	in part due to uses such as:
```
Comment (tabatkins) [inline css-ui-4/Overview.bs]: This was better as-is.

File: css-ui-4/Overview.bs [hunk 6]
```diff
@@ -1859,19 +1858,19 @@ so that CSS can be used to restyle them.
...
+		for earlier non-standard versions of this property.
 		They all have the same effect as ''appearance/auto''.
 
 		<pre class=prod style="white-space: normal"><<compat-auto>> =  <dfn>searchfield</dfn> | <dfn>textarea</dfn> | <dfn>push-button</dfn> | <dfn>slider-horizontal</dfn> | <dfn>checkbox</dfn> | <dfn>radio</dfn> | <dfn>square-button</dfn> | <dfn>menulist</dfn> | <dfn>listbox</dfn> | <dfn>meter</dfn> | <dfn>progress-bar</dfn></pre>
 
 		Issue: When ''appearance/auto'' is widely supported,
 		recommend using it instead.
 
-		Issue: If any of these value is not needed for web compat,
+		Issue: If any of these values is not needed for web compat,
```
Comment (tabatkins) [inline css-ui-4/Overview.bs]:
```suggestion
		Issue: If any of these values are not needed for web compat,
```

File: css-ui-4/Overview.bs [hunk 7]
```diff
@@ -1977,10 +1976,10 @@ Effects of 'appearance' on Semantic Aspects of Elements</h4>
 
 UAs must preserve aspects of the control
 which are necessary to operate the control with its original semantics.
-The UA may however give them a different look and feel
+The UA may however give it a different look and feel
 as long as it remains possible to operate the control.
-Aspects of a control which are merely needed to observe the state the control is in
-and are not needed to operate it
+Aspects of a control, which are merely needed to observe the state the control is in
```
Comment (tabatkins) [inline css-ui-4/Overview.bs]: these two lines were better as-is

File: css-ui-4/Overview.bs [hunk 8]
```diff
@@ -2036,8 +2035,8 @@ that are typical of the element whose appearance it acquires.
 <h3 id=control-specific-rules>
 Control Specific Rules</h3>
 
-Issue: Maybe some or all of this section
-should be moved to the [[HTML]] spec
+Issue: Maybe some part or all of this section
```
Comment (tabatkins) [inline css-ui-4/Overview.bs]: this line was better as-is

File: css-ui-4/Overview.bs (hunk 1 above)
Comment (SebastianZ) [reply css-ui-4/Overview.bs]: Right. Added.

File: css-ui-4/Overview.bs (hunk 2 above)
Comment (SebastianZ) [reply css-ui-4/Overview.bs]: Done.

File: css-ui-4/Overview.bs (hunk 3 above)
Comment (SebastianZ) [reply css-ui-4/Overview.bs]: That was new to me. Thanks!

File: css-ui-4/Overview.bs (hunk 4 above)
Comment (SebastianZ) [reply css-ui-4/Overview.bs]: Ok, added the comma back.

File: css-ui-4/Overview.bs (hunk 5 above)
Comment (SebastianZ) [reply css-ui-4/Overview.bs]: Hmm, that sounded strange to me, but you're the native speaker, so I trust you.

File: css-ui-4/Overview.bs (hunk 6 above)
Comment (SebastianZ) [reply css-ui-4/Overview.bs]: Right. Changed it.

File: css-ui-4/Overview.bs (hunk 7 above)
Comment (SebastianZ) [reply css-ui-4/Overview.bs]: Ok, changed it back.

File: css-ui-4/Overview.bs (hunk 8 above)
Comment (SebastianZ) [reply css-ui-4/Overview.bs]: Removed "part" again.

File: css-ui-4/Overview.bs [hunk 9]
```diff
@@ -707,7 +707,7 @@ Values have the following meanings:
 			If the values are unspecified,
 			then the intrinsic hotspot defined inside the image resource itself is used.
 			If both the values are unspecific and the referenced cursor has no defined hotspot,
-			the effect is as if a value of "0 0" were specified.
+			the effect is as if a value of "0 0" was specified.
```
Comment (Loirooriol) [inline css-ui-4/Overview.bs]: Not a native speaker, but this seems an unreal conditional, so "were" was correct?
https://www.grammarly.com/blog/conditional-sentences-was-instead-of-were/

Comment (ewilligers) [reply css-ui-4/Overview.bs]: I hadn't heard of 'unreal conditional' before, but I wouldn't read "0 0" as being implausible here.

As a native speaker, I would always use 'was' here due to the non-plural "a value".

The following might also have worked: "the effect is as if the values `0 0` were specified."

Comment (SebastianZ) [reply css-ui-4/Overview.bs]: I am also not a native speaker, though I looked up the grammar about unreal conditionals before I created the PR and came to the same conclusion as Eric.

The page you referred to clearly states "When you’re talking about a possibility that did happen or might be true, use “was” and “were” as you normally would."

So, according to that, "was" is correct.

Sebastian

Comment (Loirooriol) [reply css-ui-4/Overview.bs]: Well, we are under the assumption that "both the values are unspecific", so it's impossible for "0 0" to be specified. Then it seems an unreal conditional to me, but it's not like this matters much.

**Comment by SebastianZ** [comment]
Thank you for merging these changes, @frivoal! I totally missed that.

Sebastian

**Pull request merged**